test/components/common.test.js

```diff
@@ -0,0 +1,152 @@
+import { GetProtocolFlags, GetSubscriberFlags, pascalCase } from '../../components/common';
+import AsyncAPIDocument from '@asyncapi/parser/lib/models/asyncapi';
+
+const docWithoutChannels = new AsyncAPIDocument({
```

Comment:
I recommend to take these example files from test into separate files, and then just read them from file. This way you can share them in future with others tests, like integration tests